### DISCUSSION TITLE
rocketmq 5.1.3 版本 双主双从 ，从节点没有生产

### DISCUSSION THREAD

**Discussion by yifanfengshun51** [apache/rocketmq#7761] [General]
rocketmq 5.1.3 部署双主双从完成后，并且还部署了rocketmq web插件
进行数据测试 
export NAMESRV_ADDR=localhost:9876 && ./bin/tools.sh org.apache.rocketmq.example.quickstart.Producer && ./bin/tools.sh org.apache.rocketmq.example.quickstart.Consumer
然后会发现从节点根本数据
![image](https://github.com/apache/rocketmq/assets/37198490/5d126852-63cf-4910-8a74-1ede33d291c9)
但是5.0.0版本的rocketmq  从节点会有消息
![41d204b874304365128985cbbf85c16](https://github.com/apache/rocketmq/assets/37198490/21571980-027b-4682-96a7-79bc827afc5d)
这是什么原因啊？
rocketmq 5.1.3 这个版本与 rocketmq 5.0.0 版本有什么需要特别配置的地方吗

**Comment by wpy0032** [comment]
请问找到原因了吗，5.2同样的问题
